syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -393,6 +397,18 @@ def __mul__(self, other, **kwargs):
         """
         return self.mul(other, **kwargs)
 
+    def pow(self, power):
```

Comment:
FYI: There's a faster algorithm for calculating `pow` (instead of being O(power) is O(log(power)).

https://www.rookieslab.com/posts/fast-power-algorithm-exponentiation-by-squaring-cpp-python-implementation

Reply:
Thanks for the info!